Jenkinsfile_CNP

```diff
@@ -1,6 +1,6 @@
 #!groovy
 
-@Library("Infrastructure")
+@Library("Infrastructure@helm2.16.1")
```

Comment:
this should not go to master. It was just for testing.

Reply:
done